gym_roboy/envs/ros_proxy.py

```diff
@@ -128,13 +130,14 @@ def _publish_new_goal_on_rviz(self, goal_joint_angle):
         self.sphere_axis2.publish(msg2)
 
     def get_new_goal_joint_angles(self):
+        #self.node.get_logger().info("Reached goal joint angles: " + str(self.read_state().joint_angle))
         self._check_service_available_or_timeout(self.goal_client)
         req = GymGoal.Request()
         future = self.goal_client.call_async(req)
         rclpy.spin_until_future_complete(self.node, future)
         res = future.result()
         if res is not None:
-            #self.node.get_logger().info("feasible: " + str(res.q))
+            self.node.get_logger().info("feasible: " + str(res.q))
```

Comment:
The print string of this looks something like this: `"feasible: [1.0, 1.0, 1.0]"`
Does it mean that the numbers are feasible? Then one could expect some print like `"unfeasible: [70.0, 1.0, 1.0]"` but that will never occur. Are we trying to print `res.feasible` ? Do we really need this logging on every step? Maybe we can get rid of this log.

Reply:
No we don't need. But let's comment it out for future debugging.

Reply:
ok

Reply:
the only thing we really need to print is the information if the agent has reached the target state.